### PR TITLE
Validate models with exceptions

### PR DESCRIPTION
**Pull request by d2bit**
We talked about it some time ago...
It seems to me that is worth to use exceptions, so that we avoid so many repeated code `if save; else`

Let's discuss it :v:

### CODE REVIEW

**Comment by scruti** [comment]
I had doubts about it, so I searched and found articles like this one:
https://robots.thoughtbot.com/save-bang-your-head-active-record-will-drive-you-mad

http://stackoverflow.com/questions/5056385/when-to-use-save-vs-save-in-model

**Comment by d2bit** [comment]
> Exceptions should not be expected

This is (will be) an API.. So if we do validation on the client side... they are not expected :v:

I don't get the whole point on the post... I think exceptions are for exceptional cases. Is AM validation an exceptional case? I'm not sure, but I feel like for this particular case it fits well as we gonna respond the same json template to every validation error.

**Comment by scruti** [comment]
I agree :+1:

File: app/controllers/v1/events_controller.rb [hunk 1]
```diff
@@ -9,20 +9,13 @@ def show
...
+    render json: @event
   end
 
   def update
-    if @event.update_attributes(event_params)
-      render json: @event
-    else
-      render 'validation_error', status: :forbidden
-    end
+    @event.update_attributes!(event_params)
```
Comment (gponsu) [inline app/controllers/v1/events_controller.rb:17]: update_attributes! or only update!? which is better?

Comment (d2bit) [inline app/controllers/v1/events_controller.rb:17]: > you should use update_attributes , if you have a model, like this:
> book.update_attributes(params[:book])
> and you should use update, when you want on the criteria, like this:
> Book.where(id: params[:id]).update(params[:book])

https://groups.google.com/forum/#!topic/mongoid/c_bAhwtHWHs
I didn't know about it. Good question!!

Comment (gponsu) [inline app/controllers/v1/events_controller.rb:17]: Thanks! :smile:

**Pull request merged**